src/GridFieldOrderableRows.php

```diff
@@ -768,7 +786,8 @@ protected function getSortValuesFromManyManyThroughList($list, $sortField)
 
         // Find the foreign key name, ID and class to look up
         $joinClass = $manipulator->getJoinClass();
-        $fromRelationName = $manipulator->getForeignKey();
+        $foreignKeyMethod = $this->getManyManyInspectorForeignKeyMethod($manipulator);
```

Comment:
if the method returned the method invocation result, we wouldn't have to introduce this new variable here